### ISSUE TITLE
NULA version?

### ISSUE THREAD

**Issue by kieranhj** [kieranhj/pop-beeb#29]
Assuming all goes to plan, the sprite engine should handle any 3 colours from full palette of 16 per sprite.  This could look amazing using the NULA to provide 16 distinct colours from a palette of 4096.
Obviously the palette can be altered on a per level / screen basis as well.
Would need to work with Dethmunk to make the palette look amazing!

**Comment by simondotm** [comment]
Agree. Was thinking the game could ask user if they want NULA enhanced version? Y/N

**Comment by kieranhj** [comment]
Might be a separate version as the sprites will all be different.  Also going to come a way afterwards I think.

**Comment by kieranhj** [comment]
Let's say postponed to next milestone.
